apps/frontend/src/libs/components/types/breadcrumb-navigation-item.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type AppRoute } from "~/libs/enums/app-route.enum.js";
```

Comment:
Not the correct imoprt
```suggestion
import { type AppRoute } from "~/libs/enums/enums.js";
```